src/components/Tabs/Tabs.tsx

```diff
@@ -176,9 +177,10 @@ export default class Tabs extends React.PureComponent<Props, State> {
     }
 
     if (key === 'ArrowLeft' || key === 'ArrowUp') {
-      newFocus -= 1;
-      if (newFocus === -1) {
+      if (newFocus === -1 || newFocus === 0) {
```

Comment:
@tmlayton You had asked about this change.

The index defaults to `-1`. https://github.com/Shopify/polaris-react/pull/466/files#diff-78ae66000d608696759f1121a83922a4R60

When pressing either left or up from the first item, the index should become the last one (https://github.com/Shopify/polaris-react/pull/466/files#diff-78ae66000d608696759f1121a83922a4R181 but it was setting the index to `-2` if it was `-1` or to `-1` if it was `0`. In other words, it was focusing on nothing instead of the last one.